policyengine_us/parameters/gov/states/la/tax/income/credits/cdcc/non_refundable/rate.yaml

```
@@ -0,0 +1,25 @@
+description: Louisiana provides this rate of the federal cdcc as the non refundable cdcc.
+
+brackets:
+ - threshold:
```

and elsewhere. you always need two spaces for indents
```suggestion
  - threshold:
```